2016/presentations/inteligentny_dom_z_(micro)python/text.md

```diff
@@ -0,0 +1,89 @@
+# Inteligentny dom z (Micro)Python - Krzysztof Czarnota
+
+Inteligentne domy stają się coraz bardziej powszechne. Nic dziwnego, rozwiązania które ułatwiają nam codzienne życie, a także pozwalają na oszczędność czasu i pieniędzy muszą cieszyć się dużą popularnością. Na rynku pojawia się wiele firm, które oferują swoje usługi w zakresie automatyzacji budynków, ale niejednokrotnie koszty takiej instalacji odstraszają potencjalnego użytkownika. Tymczasem stworzenie systemu inteligentnego domu wcale nie musi być drogie ani trudne.
+
+## Jak zacząć?
+Instalacja inteligentnego domu składa się z sieci czujników oraz elementów wykonawczych, która monitoruje oraz kontroluje obecny stan budynku. Druga część instalacji to centralny system zarządzania, który pozwala na personalizację ustawień, a następnie na podstawie informacji z czujników steruje urządzeniami wykonawczymi. Wiele urządzeń, zwłaszcza w nowych inwestycjach, zamontowanych w budynkach pozwala na zdalne sterowanie. Jednak nie ma jeszcze jednolitego standardu, który umożliwiałby bezproblemowe połączenie i sterowanie wszystkimi urządzeniami. W naszej instalacji muszą pojawić się także elementy pozwalające na sterowanie urządzeniami, które oryginalnie nie zapewniają takiej funkcjonalności. Najprostszym rozwiązaniem obu tych problemów jest wykonanie własnego projektu sprzętowego.
```

Comment:
s/która monitoruje/która to sieć monitoruje

s/a następnie na podstawie informacji z czujników steruje/a następnie, na podstawie informacji z czujników, steruje

s/Wiele urządzeń, zwłaszcza w nowych inwestycjach, zamontowanych w budynkach pozwala na zdalne sterowanie./W budynkach (zwłaszcza w nowych inwestycjach) instalowanych jest obecnie wiele urządzeń pozwalających na zdalne sterowanie